### DISCUSSION TITLE
Trilium needs more recogniton

### DISCUSSION THREAD

**Discussion by arromattic** [TriliumNext/Trilium#4069] [Ideas]
I think trilium is far less talked about when you visit pkm or note taking subreddits or videos on youtuber
Even logseq has become lot more popular despite being being used to behind trilium in number of stars

I realized this from recently seeing a post on reddit pkm sub complaining about why trilium is unpopular .
Two complain i found was :

1. Lack of detail in Documentation problem 
2. UI is not pretty

I do feel bad about not having large number of contributors like other apps.

I still have my own question why trilium is unpopular ? 
I am loving it especially the web clipper

**Comment by XXXJetfireXXX** [comment]
1. While documentation might seem not detailed enough, people are actively working on improving it and adding all the needed information. Though, practice is still the best teacher.
2. That is a pretty subjective matter, as for me it looks nice. Not too simplistic, yet not 'overloaded', just the way I like it. Also you can use (or even make) custom css themes to further customize your experience, which is an amazing feature!

**Reply by arromattic** [reply]
Should trilium a have a theme similar to Popular note apps for more users ?

**Comment by meichthys** [comment]
The number 1 reason it is not recommended more is because there is no mobile app and the mobile web experience is very stripped down. I use Trilium daily but often hesitate to recommend it because mobile is so often a high priority when it comes to people looking for a note taking app.

**Comment by arromattic** [comment]
I am just parroting this If anyone is curious.
https://www.reddit.com/r/PKMS/comments/14mt9sg/why_is_trilium_so_unknown/

**Comment by Nriver** [comment]
I think the primary challenge that new users encounter is deploying a self-hosted server, which serves as a crucial aspect of Trilium. One must possess fundamental familiarity with Linux server operations, proficiently configure server settings, deploy applications, acquire domains, and procure HTTPS certificates.

Similar to how key shortcuts are Vim's standout feature, they also present a significant hurdle for new users.

**Reply by meichthys** [reply]
Yes this is a significant hurdle. There is trilium.cc but it would be nice if there was an official hosted instance that could handle multiple tenancy and have a small subscription cost that would all go to support Zadam's continued development of this project.

**Reply by winamp8** [reply]
maybe mobile app is very important

**Comment by 0xbismarck** [comment]
One thing that I noticed is there are not many videos talking about Trilium, and how people use it to organize there notes. Most tutorials I've seen online involve setting up the service, and what features exist. https://www.youtube.com/results?search_query=trilium+notes

I noticed for other Note taking apps, there are tutorials online for setting up things like PARA in your app - https://fortelabs.com/blog/how-to-implement-para-in-your-favorite-notetaking-app/

On the comment of Mobile apps, I noticed that complaint as a common theme in this blog post.  https://fortelabs.com/blog/test-driving-a-new-generation-of-second-brain-apps-obsidian-tana-and-mem/:

> They’ve since launched a dedicated mobile app called [Tana Capture](https://help.tana.inc/tana-capture.html), but it’s important to note that it’s only for capturing content. This is one of the common downsides of more sophisticated tools: they can’t replicate their advanced functionality in a mobile interface, forcing them to settle for a limited or handicapped set of mobile features instead. This is a major problem if mobile devices are the primary way you interact with notes.

Finally I noticed Trilium is absent on this [resource list](https://www.buildingasecondbrain.com/resources). Someone could write in to have it added.
